client/app/telecom/overTheBox/tasks/overTheBox-tasks.controller.js

```diff
@@ -14,36 +14,22 @@ angular.module("managerApp").controller("OverTheBoxTasksCtrl", function ($transl
         perPage: PAGINATION_PER_PAGE
     };
 
-    function init () {
-        self.loaders.init = true;
+    this.$onInit = function () {
         $q.all([
             self.getTasks()
-        ]).finally(function () {
-            self.loaders.init = false;
-        });
-    }
+        ]);
+    };
 
     self.getTasks = function () {
-        self.loaders.tasks = true;
-        return OvhApiOverTheBox.Lexi().getTasks({ serviceName: self.serviceName }).$promise.then(
-            function (taskIds) {
-                self.taskIds = taskIds;
-            },
-            function (error) {
-                Toast.error([$translate.instant("an_error_occured"), error.data.message].join(" "));
-            }
-        ).finally(function () {
-            self.loaders.tasks = false;
-        });
+        OvhApiOverTheBox.Lexi().getTasks({ serviceName: $stateParams.serviceName }).$promise
```

Comment:
Can we avoid mixing ES6 with ES5 ? ES6 code is not yet implemented everywhere in the TELECOM manager but it's planned. So is it the good moment to do it or the scope is only datagrid ?

Reply:
The scope is only datagrid (new code added).
@Jisay  The feedback I have is contradictory: components team wants ES6, you don't want mix, etc.